hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -590,6 +591,11 @@ public boolean rollback(final String commitInstantTime) throws HoodieRollbackExc
           .filter(instant -> HoodieActiveTimeline.EQUALS.test(instant.getTimestamp(), commitInstantTime))
           .findFirst());
       if (commitInstantOpt.isPresent()) {
+
+        LOG.info("Scheduling Rollback at instant time :" + rollbackInstantTime);
+        Option<HoodieRollbackPlan> rollbackPlan = createTable(config, hadoopConf)
```

Comment:
if this follows convention of how we do compaction scheduling, i think its good.